python/pyspark/ml/feature.py

```diff
@@ -2629,8 +2629,21 @@ class ChiSqSelector(JavaEstimator, HasFeaturesCol, HasOutputCol, HasLabelCol, Ja
     """
     .. note:: Experimental
 
-    Chi-Squared feature selection, which selects categorical features to use for predicting a
-    categorical label.
+    Creates a ChiSquared feature selector.
+    The selector supports different selection methods: `numTopFeatures`, `percentile`, `fpr`,
+    `fdr`, `fwe`.
+    `numTopFeatures` chooses a fixed number of top features according to a chi-squared test.
```

Comment:
Organize following items to make it as a list in the generated API docs, please refer https://github.com/apache/spark/blob/master/python/pyspark/ml/regression.py#L1300